packages/lit-node-client-nodejs/src/lib/helpers/get-session-key.ts

```diff
@@ -0,0 +1,64 @@
+import { EITHER_TYPE, LOCAL_STORAGE_KEYS } from '@lit-protocol/constants';
+import { generateSessionKeyPair } from '@lit-protocol/crypto';
+import { log } from '@lit-protocol/misc';
+import { getStorageItem } from '@lit-protocol/misc-browser';
+import { SessionKeyCache, SessionKeyPair } from '@lit-protocol/types';
+
+// Global cache variable
+let sessionKeyCache: SessionKeyCache | undefined = undefined;
+
+// const _expiration = new Date(Date.now() + 24 * 60 * 60 * 1000).toISOString();
+
+/**
+ * Retrieves the session key from local storage or cache, or generates a new session key if none exists.
+ * @param expiration - The expiration date of the session key.
+ * @returns The session key pair.
+ */
+export const getSessionKey = (expiration: string): SessionKeyPair => {
```

Comment:
we should rename this as to not get confused with `LitNodeClient.getSessionSigs` can we maybe add something that outlines its retrieving from cache or creating? Maybe `getOrCreateSessionKey`